name.py

```diff
@@ -0,0 +1 @@
+print('Hello, world!')
```

Comment:
Please, remove all extra files. Work should be done in the lab folder exclusively. Thank you!

Reply:
Please 🙏